recipes/pam.rb

```diff
@@ -71,7 +72,7 @@
   end
 
   # configure tally2
-  if node['os-hardening']['auth']['retries'] > 0
+  if node['os-hardening']['auth']['retries'].positive?
```

Comment:
funny thing, it looks like chef 12.5.1 was packaged with different ruby for ubuntu then to centos. I'll have a detailed look tomorrow